### DISCUSSION TITLE
how to upgrade android studio react native plugin for latest version

### DISCUSSION THREAD

**Discussion by AbdulVakeel** [community/community#114714] [Mobile] [answered]
### Select Topic Area

General

### Body

How  to upgrade android studio react native plugin for latest version

**Comment by skymlmsoftwarecompany** [comment, marked as answer]
belbuildscript {
    ext {
        // ...
        kotlin_version = '1.6.10' // <- add this line
    }

    dependencies {
        // ...
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version" // <- add this line
        // ...
    }

**Comment by AbdulVakeel** [comment]
yes its working
